### ISSUE TITLE
Unsatisfactory results

### ISSUE THREAD

**Issue by hathawayxxh** [1998v7/Self-Filtering#1]
Hi,

I am very interested in your work and I am running your code on my dataset. I find it works well when the noise rate is low (r = 0.2), but the performance is worse than the baseline model trained with CE loss when the noise rate is high (r = 0.4, 0.5). Have you ever met this before? or can you provide some suggestions on how to improve the performance?

Best,
Xiaohan

**Comment by 1998v7** [comment]
There are some suggestions I think would be useful.
1. the size of memory bank T should be set as 2 or 3.  Empirical results show that clean examples will be greatly discarded with a large T. 
2. you can try different learning rates and weight decay for mitigating fitting rate. We admit that the performance of the fluctuation criterion may be degraded if we detect fluctuation events in a noisy fine-grained dataset such as Aircraft, CUB, and some medical images. I think the primary reason is that if noise ratio is large, deep models fit noise labels so fast, and thus the results do not fluctuate.
